spec/views/layouts/listnav/_load_balancer.html.haml_spec.rb

```diff
@@ -1,6 +1,6 @@
-include QuadiconHelper
-
 describe "layouts/listnav/_load_balancer.html.haml" do
+  helper(QuadiconHelper)
```

Comment:
@chrisarcand @Fryguy @bdunne 
lol, is this the right place... are we in rspec example group class land or some magical land of rspec that is not putting the helper into the current example's test?